routes/api/comments.js

```diff
@@ -0,0 +1,38 @@
+//2 rts
+const express = require('express');
+const {asyncHandler} = require('../utils');
+const router = express.Router();
+const { CommentLike } = require("../../db/models");
+
+//
+
+//Like a comment
+router.post('/comments/:id/likes', asyncHandler(async(req, res) => {
+    const { userId, commentId } = req.body;
+    
+    const likedComment = await CommentLike.create({
+            userId: res.locals.user.id,
+            commentId: req.params.id
+
+    })
+
+    res.json(req.body);
+
+    res.redirect('/comment/:id');
+
+    
+}));
+
+
+//Unlike a comment
+router.delete('/comments/likes/:id', asyncHandler(async(req, res) => {
+        await CommentLike.destroy({
```

Comment:
Check to make sure the user is authenticated and that the user is the owner of that like (`res.locals.user.id === like.userId`)

Here is a starter:
```js
if (!res.locals.authenticated) {
   return next(createError(401)); 
}
```

Reply:
Hey, I imported the error package for all the api files and did  
(res.locals.user.id === like.userId && res.locals.authenticated) {
--CRUD--
}